### PR TITLE
Install j2 before 'make install'

### PR DESCRIPTION
**Pull request by sebrandon1**
Testing to see if this works.

### CODE REVIEW

**Review by edcdavid — APPROVED** [review]

**Comment by sebrandon1** [comment]
This needs to be merged in tandem with: 

https://github.com/test-network-function/cnf-certification-test/pull/2079

File: config/vagrant/scripts/bootstrap-docker-fedora.sh [hunk 1]
```diff
@@ -25,6 +25,12 @@ sudo dnf -y install docker-ce docker-ce-cli containerd.io jq
 
 # Configure IPv6 in docker ( https://docs.docker.com/config/daemon/ipv6/ )
 sudo mkdir /etc/docker
+
+# Create the daemon.json file if it does not exist
+if [ ! -f /etc/docker/daemon.json ]; then
+  sudo touch /etc/docker/daemon.json
+fi
```
Comment (rdavid) [inline config/vagrant/scripts/bootstrap-docker-fedora.sh:32]:
```suggestion
sudo touch -a /etc/docker/daemon.json
```

File: config/vagrant/scripts/bootstrap-docker-ubuntu.sh [hunk 2]
```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+# create the docker daemon json file if it does not exist
+if [ ! -f /etc/docker/daemon.json ]; then
+  sudo touch /etc/docker/daemon.json
+fi
```
Comment (rdavid) [inline config/vagrant/scripts/bootstrap-docker-ubuntu.sh:6]:
```suggestion
sudo touch -a /etc/docker/daemon.json
```

**Review by rdavid — APPROVED** [review]

File: config/vagrant/scripts/bootstrap-docker-fedora.sh (hunk 1 above)
Comment (sebrandon1) [reply config/vagrant/scripts/bootstrap-docker-fedora.sh:32]: Done.

**Pull request merged**